### PR TITLE
Added metrics for the root Marathon

### PR DESCRIPTION
**Pull request by meln1k**
JIRA issues: DCOS-50466

### CODE REVIEW

**Review by gracedo — CHANGES_REQUESTED** [review]
Can you make sure that you're exporting the dashboard json from grafana? ([see this comment here from another PR](https://github.com/dcos/grafana-dashboards/pull/25#discussion_r267968433))

**Comment by meln1k** [comment]
@gracedo the data sources should be good to go now!

**Comment by gracedo** [comment]
The data source is set to `null` which doesn't look quite right - can you double check that you're exporting the dashboard json directly from the Grafana UI? I would expect the diff to look something similar to [this one here](https://github.com/dcos/grafana-dashboards/pull/32/files).

**Comment by meln1k** [comment]
@gracedo yes, it was copied directly from the Grafana UI.

File: dashboards/Marathon-Root-Summary.json [hunk 1]
```diff
@@ -0,0 +1,2830 @@
...
+        {
+          "format": "decbytes",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        },
+        {
+
```
Comment (gracedo) [inline dashboards/Marathon-Root-Summary.json]: Can you modify this title to a `<Component>: <Description>` format?

**Review by gracedo — CHANGES_REQUESTED** [review]
Sorry, it's not obvious, but the `Export for sharing externally` option should be checked when exporting, that should do it! I tried it on this dashboard in SOAK and the json looks correct.
image

**Comment by meln1k** [comment]
@gracedo Alright, it is fixed by now :)

**Review by gracedo — APPROVED** [review]
Lgtm thanks!!

**Pull request merged**